template/src/index.tsx

```diff
@@ -23,47 +23,55 @@ import Sidebar from './components/Sidebar';
 import Field from './components/Field';
 import Dialog from './components/Dialog';
 
-init((sdk: BaseExtensionSDK) => {
+import LocalhostWarning from './components/LocalhostWarning';
+
+if (process.env.NODE_ENV === 'development' && window.self === window.top) {
```

Comment:
Do we care if we're not in Dev mode? i.e: before we'd redirect even when not in dev mode

Reply:
Otherwise - it looks good to me